### DISCUSSION TITLE
Xiaomi ZNCZ04LM exposes "illuminance" and "illuminance_lux"

### DISCUSSION THREAD

**Discussion by Allineer** [Koenkk/zigbee2mqtt#13284] [Devices]
z2m [1.26.0](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.26.0) commit: [cd23aea](https://github.com/Koenkk/zigbee2mqtt/commit/cd23aea)

zStack12

Xiaomi ZNCZ04LM (lumi.plug.mmeu01)
```
{
    "auto_off": true,
    "consumer_connected": true,
    "consumption": 610.95,
    "current": 0.1954,
    "device_temperature": 30,
    "energy": 610.95,
    "illuminance": 0,
    "illuminance_lux": 0,
    "last_seen": 1658774568931,
    "led_disabled_night": false,
    "linkquality": 52,
    "power": 37.54,
    "power_outage_count": 99,
    "power_outage_memory": false,
    "state": "ON",
    "temperature": 27,
    "update": {
        "state": "idle"
    },
    "update_available": false,
    "voltage": 227
}
```

What is "illuminance" and "illuminance_lux"? should it be here?

**Comment by Allineer** [comment]
PS: what's is the "device_temperature" and "temperature"?

**Reply by Otnow** [reply]
`device_temperature` is a service value of the temperature of the chip inside the device, which was originally called simply `temperature`, but in order not to be confused with the _temperature_ value of climate sensors, it was renamed to `device_temperature`.
Accordingly, you can remove the unused `temperature` state from the _state.json_ file.

**Comment by Otnow** [comment]
> What is "illuminance" and "illuminance_lux"? should it be here?

No, it shouldn't be like this, so I created a [PR](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4485) to fix it.

**Comment by Allineer** [comment]
Thanks,  @Otnow!
